src/Common/tests/TestUtilities/System/RetryHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
I only brought in the history from the `src/Microsoft.Bcl.HashCode/` directory in corefx. This file was not part of that folder so I had to add it manually.